Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -115,8 +115,8 @@ IOS::ES::TicketReader CVolumeWiiCrypted::GetTicket() const
 
 IOS::ES::TMDReader CVolumeWiiCrypted::GetTMD() const
 {
-  u32 tmd_size;
-  u32 tmd_address;
+  u32 tmd_size = 0;
+  u32 tmd_address = 0;
```

Comment:
Instead of making Dolphin try to read a TMD from address 0, you should return an empty `TMDReader` if either of the `ReadSwapped` calls fail.